testsuite/forge/src/tx_emitter.rs

```diff
@@ -0,0 +1,717 @@
+        }
+        info!("Completed creating seed accounts");
+
+        Ok(seed_accounts)
+    }
+
+    pub async fn mint_accounts(
+        &mut self,
+        req: &EmitJobRequest,
+        requested_accounts: usize,
+    ) -> Result<()> {
+        if self.accounts.len() >= requested_accounts {
+            info!("Not minting accounts");
+            return Ok(());
+        }
+        let num_accounts = requested_accounts - self.accounts.len(); // Only minting extra accounts
+        let coins_per_account = SEND_AMOUNT * MAX_TXNS;
+        let coins_total = coins_per_account * num_accounts as u64;
+        let tx_factory = self.tx_factory.clone();
+        let client = self.pick_mint_client(&req.json_rpc_clients);
+
+        // Create seed accounts with which we can create actual accounts concurrently
+        let seed_accounts = self
+            .get_seed_accounts(&req.json_rpc_clients, req.json_rpc_clients.len())
+            .await?;
+        let faucet_account = self.get_money_source(coins_total).await?;
+        let actual_num_seed_accounts = seed_accounts.len();
+        let num_new_child_accounts =
+            (num_accounts + actual_num_seed_accounts - 1) / actual_num_seed_accounts;
+        let coins_per_seed_account = coins_per_account * num_new_child_accounts as u64;
+        mint_to_new_accounts(
+            faucet_account,
+            &seed_accounts,
+            coins_per_seed_account as u64,
+            100,
+            client.clone(),
+            tx_factory.clone(),
+        )
+        .await
+        .map_err(|e| format_err!("Failed to mint seed_accounts: {}", e))?;
+        println!("Completed minting seed accounts");
+        println!("Minting additional {} accounts", num_accounts);
+
+        // For each seed account, create a future and transfer diem from that seed account to new accounts
+        let account_futures = seed_accounts
+            .into_iter()
+            .enumerate()
+            .map(|(i, seed_account)| {
+                // Spawn new threads
+                let index = i % req.json_rpc_clients.len();
+                let cur_client = req.json_rpc_clients[index].clone();
+                create_new_accounts(
+                    seed_account,
+                    num_new_child_accounts,
+                    coins_per_account,
+                    20,
+                    cur_client,
+                    tx_factory.clone(),
+                )
+            });
+        let mut minted_accounts = try_join_all(account_futures)
+            .await
+            .map_err(|e| format_err!("Failed to mint accounts {}", e))?
+            .into_iter()
+            .flatten()
+            .collect();
+
+        self.accounts.append(&mut minted_accounts);
+        assert!(
+            self.accounts.len() >= num_accounts,
+            "Something wrong in mint_account, wanted to mint {}, only have {}",
+            requested_accounts,
+            self.accounts.len()
+        );
+        println!("Mint is done");
```

Comment:
Is there something stopping us from using the logger instead of `println!`?  I know some portions of our testing don't play well with it / then it doesn't show up in CI.

Reply:
I did use info! but it seems can not print out anything, so I did not dig into this details and just used println! instead as temp solution, because I have planned another pr with stats report, which I will do all those sort of stuff together with logs